infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesPersistentVolumeClaims.java

```diff
@@ -158,6 +158,26 @@ public void delete(Map<String, String> labels) throws InfrastructureException {
     }
   }
 
+  /**
+   * Removes PVC by name.
+   *
+   * @param name the name of the PVC
+   * @throws InfrastructureException
```

Comment:
please add a few words

Reply:
there are a few words already ;-)

Reply:
oh, you probably mean smth. like `when any error occurs while removing` for exeption

Reply:
fixed